### ISSUE TITLE
Can we save the JSON output?

### ISSUE THREAD

**Issue by kadimulam** [jdorn/json-editor#577]
Once the JSON is edited, can we save the final JSON to disk? I have seen the basic demo, but it does not have an option to save. Could you please clarify?

**Comment by TrevorPage** [comment]
This would be completely out of scope of the library. This would need to be handled with your own code, possibly using another library such as https://github.com/eligrey/FileSaver.js.

**Comment by kadimulam** [comment]
I would like to save at the server side, not at the client side. Can i read the contents of the "JSON Output" text area and save to disk (at the server side)?

**Comment by TrevorPage** [comment]
Yes. The resultant output is simply a JavaScript object which you could read and simply POST or PUT back to your server.

**Comment by apena415** [comment]
How would one access the contents of the "JSON Output"?

**Comment by fabiocbinbutter** [comment]
https://github.com/jdorn/json-editor#getset-value
